index.html

```diff
@@ -104,6 +105,7 @@ <h2 class="main-heading">I'm Shaili<br />Glad to see you!</h2>
       </section>
       <!-- #### Project-Work-section #### -->
       <section class="projects" id="portfolio">
```

Comment:
- [ ] Kindly delete the hardcoded projects, this section requires the project to be created dynamically with Javascript.